spec.md

```diff
@@ -320,6 +329,24 @@ how platforms might expose these values to their users.
 
 After implementing the first endpoint `GET /v2/catalog` documented [above](#catalog-management), the service broker will need to be registered with your platform to make your services and plans available to end users.
 
+### Deprecating services and plans
+
+While the names and plans of services are technically mutable, changing the
+names of these entities is not supported uniformly on all platforms and might
+not provide a good experience for users of all platforms.  In order to provide
+users with the best experience when a service or plan name changes, broker
+authors SHOULD use the `deprecated` field to indicate that a service or plan
+will be removed in the future and create a new service or plan with the new
+name.
+
+Platforms MUST allow existing instances of deprecated services or on
+deprecated plans to be updated (for example, to change configuration
+parameters or update to a non-deprecated plan).
+
+Platforms MAY not allow new instances of deprecated services or plans to be
```

Comment:
"MAY not" is a bit awkward.  Perhaps:
While creating new uses of deprecated services or plans is allowed, it is RECOMMENDED that platforms discourage their users from doing so.